src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -34,12 +39,24 @@ public DeleteCommand(Index targetIndex) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Person> lastShownList = model.getFilteredPersonList();
+        List<Task> lastShownTaskList = model.getFilteredTaskList();
 
         if (targetIndex.getZeroBased() >= lastShownList.size()) {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
+
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
+
+        for (Task task : lastShownTaskList) {
+            if (task.getPersonEmailAddress().equals(personToDelete.getEmail())) {
+                Task editedTask = task.copy();
+                editedTask.setPersonName(new Name("The person assigned to this task has been deleted"));
```

Comment:
Should we just set this to the no person assigned?

Reply:
Name of person changed to "No person assigned"